src/machine.rs

```diff
@@ -33,6 +33,8 @@ use crate::{
 
 // Some global facts about the emulated machine.
 pub const PAGE_SIZE: u64 = 4 * 1024; // FIXME: adjust to target architecture
+/// The number of the available real-time signal with the lowest priority; the number should be between 32 and 64
```

Comment:
This comment is not really helpful unfortunately. Miri doesn't do anything with signals. So what consequences does changing this have? Or is it just as arbitrary as the stack size?

Reply:
Pretty much. It's just a value read by code and used to compute arguments to other platform specific functions.